iothub/service/tests/Messaging/MessageClientTests.cs

```diff
@@ -70,10 +70,7 @@ public async Task MessagesClient_SendAsync_WithModule_NullDeviceIdThrows()
         {
             // arrange
             string payloadString = "Hello, World!";
-            byte[] payloadBytes = Encoding.UTF8.GetBytes(payloadString);
```

Comment:
It didn't matter for these tests, but the payload should have been left as the intended object.